.github/workflows/build.yml

```diff
@@ -36,6 +36,21 @@ jobs:
       with:
         go-version: ^1.19.9
       id: go
+    - name: Cache drivers
+      uses: actions/cache@v3
+      with:
+        # In order:
+        # * Driver for linux
+        # * Driver for macOS
+        # * Driver for windows
+        path: |
+          ~/.cache/ms-playwright-go
+          ~/.cache/ms-playwright
+          ~/Library/Caches/ms-playwright-go
+          ~/Library/Caches/ms-playwright
+          ~\AppData\Local\ms-playwright-go
+          ~\AppData\Local\ms-playwright
+        key: ${{ runner.os }}-go-${{ matrix.browser }}-${{ hashFiles('**/package.json') }}
```

Comment:
Hashing the package.json feels wrong here, I guess it should be run.go instead?

Reply:
actions/setup-go@v4 already caches go modules.  actions/cache is only used to cache the playwright driver and browsers, so only the playwright version in `package.json` is required.

Reply:
but which package.json file do you target? Since this project only has a package.json inside the driver, aka. the ms-playwright-go directory if I remember correctly.

Reply:
Oh, I misunderstood, thought `cache/msplaywrigh-go/package.json` was referenced here. change it now.